liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/DevTest.java

```diff
@@ -160,7 +160,21 @@ public void manualTestsInvocationTest() throws Exception {
       assertTrue(verifyLogMessageExists("Unit tests finished.", 10000));
       assertTrue(verifyLogMessageExists("Integration tests finished.", 2000));
    }
-   
+
+   @Test
+   public void restartServerTest() throws Exception {
+      String GENERATE = "Running liberty:generate-features";
```

Comment:
This string is already declared in [BaseDevTest](https://github.com/OpenLiberty/ci.maven/blob/c000a5973b5dbb475a68327d37bcb6f6409a12bc/liberty-maven-plugin/src/it/dev-it/src/test/java/net/wasdev/wlp/test/dev/it/BaseDevTest.java#L65)

Reply:
Thanks, I only searched in the current file.